doc/guacamole-branding-example/guac-manifest.json

```diff
@@ -29,6 +29,7 @@
     },
     
     "translations" : [
-        "translations/en.json"
+        "translations/en.json",
+		"translations/fr.json"
```

Comment:
Two issues, here:
1. Please make sure you use 4-space tabs, in keeping with the standards of the rest of the code.
2. You added the fr.json entry, here, but there is no fr.json file added to this. There's no reason to add entries if you're not going to add the translation file.